docs/contributor/testing-strategy.md

```diff
@@ -58,6 +58,7 @@ A test plan is a document outlining the objectives, resources, and procedures fo
 
 | Name | Description                                                                                                                                                                              |
 |------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [image-build-pr](https://github.com/kyma-project/cloud-manager/blob/main/.github/workflows/image-build-pr.yaml)| Building the image. For configuration check [image-builder.yml](https://raw.githubusercontent.com/kyma-project/test-infra/refs/heads/main/.github/workflows/image-builder.yml).  
```

Comment:
```suggestion
| [image-build-pr](https://github.com/kyma-project/cloud-manager/blob/main/.github/workflows/build-image-pr.yaml)| Building the image. For configuration check [image-builder.yml](https://raw.githubusercontent.com/kyma-project/test-infra/refs/heads/main/.github/workflows/image-builder.yml).  
```

Reply:
Did you mean `https://github.com/kyma-project/cloud-manager/blob/main/.github/workflows/build-image-pr.yaml` ?